torch/fx/experimental/fx_acc/acc_normalizer.py

```diff
@@ -243,26 +235,38 @@ def move_kwargs_to_acc_out_ty(
     # acc_out_ty instead.
     tmd_dict: Dict[str, Any] = {}
 
-    # Extract quantization related kwargs
-    qscheme_to_fields = {
-        torch.per_tensor_affine: {"scale", "zero_point", "dtype"},
-        torch.per_channel_affine: {"scale", "zero_point", "axis", "dtype"},
-    }
-    qscheme = normalization_info.qscheme
-    qparams = None
-    if qscheme is not None:
-        qparams = {}
-        qparams["qscheme"] = qscheme
-    qparam_fields = qscheme_to_fields[qscheme] if qscheme is not None else set()
-    for (
-        orig_kwarg_name,
-        tmd_field_name,
-    ) in normalization_info.kwargs_to_move_to_acc_out_ty:
-        if tmd_field_name in qparam_fields:
-            qparams[tmd_field_name] = new_kwargs[orig_kwarg_name]
+    for kwarg_replacement_tuple in normalization_info.kwargs_to_move_to_acc_out_ty:
+        if len(kwarg_replacement_tuple) == 2:
+            orig_kwarg_name, tmd_field_name, is_constant = *kwarg_replacement_tuple, False
+        else:
+            assert len(kwarg_replacement_tuple) == 3
+            orig_kwarg_name, tmd_field_name, is_constant = kwarg_replacement_tuple
+        if is_constant:
+            # when the arg is constant, the fields will be name and value
+            tmd_dict[orig_kwarg_name] = tmd_field_name
```

Comment:
Can we make the tuple (constant, tmd_field_name, is_constant) so that here we can do `tmd_dict[tmd_field_name) = orig_kwarg_name`?

Reply:
oh OK